algebird-test/src/test/scala/com/twitter/algebird/SpaceSaverTest.scala

```diff
@@ -26,22 +31,31 @@ class SpaceSaverLaws extends CheckProperties {
           Equiv.fromFunction { (left, right) =>
             (left consistentWith right) && (right consistentWith left)
           }
+
         commutativeSemigroupLaws[SpaceSaver[Int]]
       }
     }
   }
+
+  property("SpaceSaver can serialize/deserialize itself") {
+    forAll { ss: SpaceSaver[String] =>
```

Comment:
Can we also add a test that exercises Failure? Like pass an empty array, and see that you get Failure not an exception thrown.

So basically: `forAll { a: Array[Byte] => ` deserialization does not throw? You get Success or Failure.